src/screens/ChannelListScreen/elements/components/AvatarPostNotif.js

```diff
@@ -26,7 +27,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     flex: 1,
     borderWidth: 1,
-    borderColor: 'white'
+    borderColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color to a theme color is good for consistency and maintainability. However, it's important to ensure that `COLORS.white` is defined in the theme and that it corresponds to the color previously used ('white'). If not, this could lead to unexpected visual changes.
```diff
-    borderColor: 'white'
+    borderColor: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->